msal/build.gradle

```diff
@@ -308,16 +308,17 @@ task pmd(type: Pmd) {
 afterEvaluate {
     // Gradle 6.1.1 and Android Gradle Plugin 4.0.1 doesn't rename the file (see 'outputFileName' above)
     // Adding this work around to have the file properly renamed.
-    tasks.named("assembleDistRelease").configure {
-        def buildFile = file("$buildDir/outputs/aar/${archivesBaseName}-dist-release.aar")
-        print "Build file $buildFile"
-        doLast {
-            println "Renaming build file $buildFile to '$buildDir${File.separator}outputs${File.separator}aar${File.separator}${archivesBaseName}-${version}.aar'"
-            if (!buildFile.renameTo("$buildDir${File.separator}outputs${File.separator}aar${File.separator}${archivesBaseName}-${version}.aar")) {
-                println "Rename failed!"
-            }
-        }
-    }
+    // Please uncomment the section below for the Maven release step only. Currently troubleshooting why this extra renaming configuration is still needed for MSAL.
+//    tasks.named("assembleDistRelease").configure {
+//        def buildFile = file("$buildDir/outputs/aar/${archivesBaseName}-dist-release.aar")
+//        print "Build file $buildFile"
+//        doLast {
+//            println "Renaming build file $buildFile to '$buildDir${File.separator}outputs${File.separator}aar${File.separator}${archivesBaseName}-${version}.aar'"
```

Comment:
>archivesBaseName [](http://example.com/codeflow?start=130&length=16)

I don't see this is done the similar way in common.

Reply:
I've been trying to wrap my head around this as well. For some reason, the usual rename config outside of this afterEvaluate block (which common does have) doesn't work for MSAL, even though it works for common when doing the maven release. I wasn't around when this block was originally added in, so I don't have the context, but somehow it was ok to be present for both vsts and maven release processes. But with the latest changes I've made to this file, we'll have to comment and uncomment it, until we can figure out the difference between MSAL and common.